packages/nitro-protocol/src/contract/transaction-creators/force-move.ts

```diff
@@ -10,7 +10,7 @@ import {getFixedPart, getVariablePart, hashAppPart, State} from '../state';
 // TODO: Currently we are setting some arbitrary gas limit
 // To avoid issues with Ganache sendTransaction and parsing BN.js
 // If we don't set a gas limit some transactions will fail
-const GAS_LIMIT = 3000000;
+const GAS_LIMIT = 100_000;
```

Comment:
This seems too low to me. The gas scales with the number of states submitted and the complexity of the `ForceMoveApp.validTransition`.  One our test cases already used 137542 gas.

Questions: 

1. What is the downside of having a conservative `GAS_LIMIT` with a lot of headroom?
2. Can we use something to estimate an appropriate `GAS_LIMIT` and not use a constant? EVM-emulation?

Reply:
https://28593-209829093-gh.circle-artifacts.com/0/nitro-protocol-gas.json

Reply:
These are just placeholders. We need to replace them with reasonable amounts for each function. I recommend using `max` from the above and filling them in for each. Ideally we could use some sort of gas estimator that depends on the complexity of the app's valid transition function.